www/src/components/markdown-page-footer.js

```diff
@@ -3,31 +3,27 @@ import { jsx } from "theme-ui"
 import React from "react"
 import { graphql } from "gatsby"
 import EditIcon from "react-icons/lib/md/create"
+import useLocalizedStrings from "./use-localized-strings"
 
-export default class MarkdownPageFooter extends React.Component {
-  constructor() {
-    super()
-    this.state = { feedbackSubmitted: false }
```

Comment:
I don't know what this was used for. Text searching the rest of www yielded no results, so I'm assuming it's something old or copy-pasta'd.

Reply:
There used to be a "Was this doc helpful" element in the footer of each page, it was replaced by the hovering widget in the right bottom corner. So I'd say it's save to delete, yeah.